packages/commonwealth/client/styles/modals/order_topics_modal.scss

```diff
@@ -1,25 +1,30 @@
-@import 'client/styles/shared';
+@import 'shared';
 
 .OrderTopicsModal {
+  width: 360px;
+
   .featured-topic-list {
-    .cui-list-item {
-      cursor: default;
-      pointer-events: none;
+    display: flex;
+    flex-direction: column;
+    height: 320px;
+    margin-bottom: 16px;
+    overflow-y: auto;
+
+    @include visibleScrollbar(light);
+
+    .topic-row {
+      align-items: center;
+      display: flex;
+      justify-content: space-between;
+      padding: 8px;
+
       &:hover {
-        cursor: default;
-        background-color: $white;
+        background-color: $neutral-50;
       }
+
       &.gu-transit {
         background-color: $neutral-100;
       }
-      .cui-icon {
-        padding: 2px;
-        cursor: pointer;
```

Comment:
Is there a reason you stripped the cursor styling from the topic ordering modal? (Here and above, with the cursor: default)

Probably should be some UI indication that a user can see to know where to drag the element. Cursor pointer seems like a good way to cue that.